gradle/gradle-mvn-push.gradle

```diff
@@ -120,7 +120,9 @@ def androidArtifactTasks() {
         source = android.sourceSets.main.java.srcDirs
 
         classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-        exclude '**/internal/*'
```

Comment:
This exclusion causes a bunch of errors during the javadoc task.  Also there is an interface to a RidesDeeplink button in here that should either get removed or moved out of internal.